examples/buildpacks/README.md

```diff
@@ -50,3 +50,4 @@ cosign verify-attestation --key k8s://tekton-chains/signing-secrets "${DOCKER_IM
 ## Links
 
 * Buildpacks: <https://buildpacks.io/>
+
```

Comment:
This seems to be an extra line left over from a previous edit.

Reply:
I believe this was semi-intentional to prevent the "No new line at end of file" error. But to be fair it is added automatically by editor config.